DESCRIPTION

```diff
@@ -26,6 +30,7 @@ Imports:
 	mlr3measures (>= 0.3.1),
 	mlr3learners,
 	mlr3pipelines,
+	mlr3viz,
```

Comment:
There are some additional packages we need to import here such as  ggparty, ggally etc. -> Can we add them?